scalameta/tokenizers/shared/src/main/scala/scala/meta/tokenizers/TokenizerOptions.scala

```diff
@@ -4,6 +4,7 @@ import scala.util.DynamicVariable
 
 final class TokenizerOptions(
     // options which control how tokens are emitted
+    val groupWhitespace: Boolean = true
```

Comment:
same question as above, could it be an opt-in rather than an opt-out?
```suggestion
    val groupWhitespace: Boolean = false
```